core/modules/qana/QservRestrictorPlugin.cc

```diff
@@ -601,37 +580,29 @@ QservRestrictorPlugin::applyLogical(query::SelectStmt& stmt,
     // Prepare to patch the WHERE clause
     query::WhereClause& wc = stmt.getWhereClause();
 
-    std::shared_ptr<query::QsRestrictor::PtrVector const> restrsPtr = wc.getRestrs();
     query::AndTerm::Ptr originalAnd(wc.getRootAndTerm());
-    query::QueryContext::RestrList restrictors;
-    // Now handle the explicit restrictors
-    if (restrsPtr && not restrsPtr->empty()) {
+    query::QueryContext::RestrList ctx_restrictors;
```

Comment:
Ok I switch case, but FYI I can't find this requirement in LSST C++ coding standards. Furthermore, using non-camelcase for naming local variables could increase readability, isn't it?